docs/client-sdk/javascript/tutorials/label-conversations.md

```diff
@@ -5,13 +5,19 @@ sidebar_position: 4
 
 # Use labels to differentiate multiple conversations between a pair of addresses
 
-With XMTP, a pair of blockchain account addresses can have multiple ongoing conversations. In addition, with XMTP, an app can display all conversations a user has, regardless of the app they used to create the conversation. This concept is known as a portable, or universal, inbox.
+With XMTP, a pair of blockchain account addresses can have multiple ongoing conversations. In addition, with XMTP, an app can display all conversations a user has, regardless of the app they used to create the conversation. This concept is known as an interoperable inbox.
 
 Alternatively, an app can implement conversation IDs and metadata to [filter conversations](filter-conversations). For example, a developer can choose to assign app-specific conversation IDs and metadata to filter and display only conversations created by their app.
 
-Apps that assign conversation IDs in this way have an effect on apps that use a portable inbox model. For example, a user might see multiple separate and ongoing conversations they are having with the same address, but originating in different apps.
+Apps that assign conversation IDs in this way have an effect on apps that use an interoperable inbox model. For example, a user might see multiple separate and ongoing conversations they are having with the same address, but originating from contexts with different conversation IDs. For example, the conversations may have originated in different apps or may have different subject lines.
```

Comment:
@yash-luna Such a great callout:

>How about 'but with different conversationIds' instead of 'but originating in different apps'?
>
>Trying to think of how we can remove the implication that chats originating in different apps will/should have a different conversationId.

How about this edit?

Reply:
Sorry for being nit-picky but can we focus away from 'app origin' as it relates to conversationId? Today, convos originating different apps (Orb, Lenster, Butterfly, even Converse) have the same 'Lens' conversationId. 

'app origin' can be reflected via conversationId (not verifiably -> non-Lens apps can use Lens conversationId) but conversationId serves a slightly different goal - i.e adding context to conversations

Reply:
Ah, I see - this helps so much!

>conversationId serves a slightly different goal - i.e adding context to conversations

Reply:
And no sorries necessary and certainly NOT nit-picky. <3 I appreciate striving for clarity - and more often than not - it requires more than a quick one-way comment on a PR!  So I appreciate your willingness to engage in a conversation to help me get to that clarity!